<ui version="4.0" >
 <class>controllerWidget</class>
 <widget class="QWidget" name="controllerWidget" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>120</width>
    <height>119</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" >
   <property name="spacing" >
    <number>1</number>
   </property>
   <property name="leftMargin" >
    <number>1</number>
   </property>
   <property name="topMargin" >
    <number>1</number>
   </property>
   <property name="rightMargin" >
    <number>1</number>
   </property>
   <property name="bottomMargin" >
    <number>1</number>
   </property>
   <item>
    <layout class="QVBoxLayout" >
     <property name="spacing" >
      <number>1</number>
     </property>
     <item>
      <layout class="QGridLayout" >
       <property name="topMargin" >
        <number>1</number>
       </property>
       <property name="horizontalSpacing" >
        <number>1</number>
       </property>
       <property name="verticalSpacing" >
        <number>3</number>
       </property>
       <item row="0" column="0" >
        <widget class="QLabel" name="label" >
         <property name="text" >
          <string>Шаги</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1" >
        <widget class="QSpinBox" name="stepEdit" >
         <property name="minimum" >
          <number>1</number>
         </property>
         <property name="maximum" >
          <number>10000</number>
         </property>
         <property name="value" >
          <number>360</number>
         </property>
        </widget>
       </item>
       <item row="1" column="0" >
        <widget class="QLabel" name="label_2" >
         <property name="text" >
          <string>Периоды</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1" >
        <widget class="QSpinBox" name="countEdit" >
         <property name="minimum" >
          <number>1</number>
         </property>
        </widget>
       </item>
       <item row="2" column="0" colspan="2" >
        <widget class="QPushButton" name="buildButton" >
         <property name="text" >
          <string>Построить график</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0" colspan="2" >
        <widget class="QProgressBar" name="progressIndicator" >
         <property name="enabled" >
          <bool>false</bool>
         </property>
         <property name="value" >
          <number>0</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
